packages/web-components/fast-components-msft/src/slider/slider.styles.ts

```diff
@@ -17,10 +17,6 @@ import {
 } from "../styles";
 
 export const SliderStyles = css`
-    :host([hidden]) {
-        display: none;
-    }
```

Comment:
Was this removal intended?

Reply:
Yes, the `${display("inline-grid")}` below already handles the `hidden` selector, right?

Reply:
Would you look at that 🤣 I did not know that. Thanks for the attention to detail!